pkg/svc/zserver.go

```diff
@@ -8,12 +8,7 @@ import (
 )
 
 // NewBasicServer constructs a new BasicServer and connects to a postgres db
-func NewBasicServer(dsn string) (pb.ContactsServer, error) {
-	db, err := gorm.Open("postgres", dsn)
-	if err != nil {
-		return nil, err
-	}
-
+func NewBasicServer(db *gorm.DB) (pb.ContactsServer, error) {
```

Comment:
To match the output of the bootstrap tool more closely, I would suggest a tweak to the comment and argument name
```
// NewBasicServer returns an instance of the default server interface
func NewBasicServer(database *gorm.DB) (pb.ContactsServer, error) {
	return & pb.ContactsDefaultServer{db: database}, nil
}
```